addons/medical_statemachine/stringtable.xml

```diff
@@ -1,99 +1,69 @@
-            <Key ID="STR_ACE_Medical_statemachine_enableUnconsciousnessAI_DisplayName">
-                <English>AI Unconsciousness</English>
-                <Russian>Потеря сознания ботами</Russian>
-                <Polish>Nieprzytomność AI</Polish>
-                <Spanish>Inconsciencia IA</Spanish>
-                <German>KI-Bewusstlosigkeit</German>
-                <Czech>Bezvědomí AI</Czech>
-                <Portuguese>Inconsciência da IA</Portuguese>
-                <French>Inconscience des IA</French>
-                <Hungarian>AI eszméletlenség</Hungarian>
-                <Italian>Incoscienza IA</Italian>
-                <Japanese>AI の気絶</Japanese>
-                <Korean>인공지능 기절</Korean>
-                <Chinesesimp>AI无意识</Chinesesimp>
-                <Chinese>AI無意識</Chinese>
-            </Key>
-            <Key ID="STR_ACE_Medical_statemachine_enableUnconsciousnessAI_Description">
-                <English>Allow AI to go unconscious</English>
-                <Russian>Позволить ботам терять сознание</Russian>
-                <Polish>Czy AI może być nieprzytomne od odniesionych obrażeń?</Polish>
-                <Spanish>Permita a la IA caer inconsciente</Spanish>
-                <German>KI kann bewusstlos werden</German>
-                <Czech>Umožňuje AI upadnout do bezvědomí</Czech>
-                <Portuguese>Permite IA ficar inconsciente</Portuguese>
-                <French>Autoriser les IA à tomber inconscients</French>
-                <Hungarian>Engedélyezi az AI eszméletének elvesztését</Hungarian>
-                <Italian>Permetti alle IA di diventare incoscienti</Italian>
-                <Japanese>AI が気絶をするようになります</Japanese>
-                <Korean>인공지능도 기절에 빠지게 합니다</Korean>
-                <Chinesesimp>允许AI进入无意识状态</Chinesesimp>
-                <Chinese>允許AI進入無意識狀態</Chinese>
-            </Key>
-        </Container>
+        <Key ID="STR_ACE_Medical_Statemachine_SubCategory">
+            <English>States</English>
+            <German>Zustände</German>
+            <Japanese>状態</Japanese>
+            <Russian>Состояния</Russian>
+        </Key>
+        <Key ID="STR_ACE_Medical_Statemachine_FatalInjuryCondition_DisplayName">
+            <English>Fatal Injury Player</English>
+            <German>Tödliche Spielerverletzungen</German>
+            <Japanese>致命傷プレイヤー</Japanese>
+            <Russian>Смертельные ранения игрока</Russian>
+        </Key>
+        <Key ID="STR_ACE_Medical_Statemachine_FatalInjuryCondition_Description">
+            <English>Controls when a player can receive a fatal injury.</English>
+            <German>Definiert wann ein Spieler tödliche Verletzungen bekommen kann</German>
+            <Japanese>プレイヤーが致命傷から蘇生できるかどうか決定します</Japanese>
+            <Russian>Определяет, когда игрок может получить смертельное ранение</Russian>
+        </Key>
+        <Key ID="STR_ACE_Medical_Statemachine_FatalInjuryConditionAI_DisplayName">
+            <English>AI Instant Death</English>
```

Comment:
I think this is too ambiguous, it's also weird how this setting only does anything if AI unconsciousness is enabled (because otherwise they always die instantly upon receiving a wound which would push them into unconsciousness).

Reply:
Could actually be a 3 way combination of both settings:
- AI can't go unconscious
- AI can go unconscious
- AI can receive a fatal injury (and go unconscious)

Just hard to find an appropriate title/wording

Reply:
🤔 Maybe it's actually fine as is, this is a really annoying setting design pattern, but I guess the unconscious and fatal injury systems are separate ultimately.

Would still reword this so that it's clear user is enabling/disabling fatal injuries for AI. That way when we have a section in the docs explaining fatal injuries it's clearly related to this setting.